app/src/androidTest/java/com/android/unio/ui/association/AssociationProfileTest.kt

```diff
@@ -102,9 +111,34 @@ class AssociationProfileTest {
           onSuccess(events)
         }
 
+    every { concurrentAssociationUserRepository.updateFollow(any(), any(), any(), any()) } answers
+        {
+          val onSuccess = args[2] as () -> Unit
+          onSuccess()
+        }
+
     every { userRepository.init(any()) } just runs
     userViewModel = UserViewModel(userRepository)
-    val user = MockUser.createMockUser()
+    val user =
+        User(
+            uid = "1",
+            email = "",
+            firstName = "",
+            lastName = "",
+            biography = "",
```

Comment:
I cannot use MockUser because it use a "MockReferenceList" as list of followed association and they are no implementation of interaction on this list, which is problematic in my test